utils/zerovec/src/yoke_impls.rs

```diff
@@ -263,58 +263,57 @@ where
     }
 }
 
-impl<'a, T: 'static + AsULE + ?Sized> ZeroCopyFrom<'a, ZeroVec<'_, T>> for ZeroVec<'a, T> {
+impl<'zcf, T> ZeroCopyFrom<'zcf, ZeroVec<'_, T>> for ZeroVec<'zcf, T>
+where
+    T: 'zcf + AsULE + ?Sized,
```

Comment:
issue: not sure if `T: 'zcf` makes sense here, but aiui it doesn't cause harm to have

Reply:
We need either `T: 'zcf` or `T: 'static` to make the compiler happy when compiling `ZeroVec<'zcf, T>`, so I figured we should use the less restrictive lifetime.

Reply:
Yeah, I think it should be `'static` for now, because a non-`'static` AsULE doesn't quite make sense (`.get()` can't work), and it seems misleading to have a `'zcf` bound that isn't actually useful.

Reply:
ok, done.